apps/web/src/components/sharing/ShareDialog.test.tsx

```
@@ -26,33 +30,85 @@ const folder: ListingRow = {
 
   const facade = {
     subscribe: (_listener: (event: EventDescriptor) => void) => () => undefined,
     snapshot: () => new Promise<never>(() => undefined),
     setFocus: () => Promise.resolve(),
-    importContact: vi.fn((code: Uint8Array) =>
-      answer('importContact', {
-        kind: 'contactImported' as const,
-        identityPublicKey: new Uint8Array(33).fill(code[0] ?? 1),
-        encPublicKey: new Uint8Array(32).fill(1),
+    sharing: vi.fn(
+      (scope: Uint8Array): Promise<SharingDescriptor> =>
```

ESLint

[error] Delete `⏎······`

(prettier/prettier)